app/models/chargeback.rb

```diff
@@ -170,4 +170,8 @@ def self.get_report_time_range(options, interval, tz)
   def self.report_cb_model(model)
     model.gsub(/^Chargeback/, "")
   end
+
+  def self.chargeback_model?(model)
+    %w(ChargebackContainerProject ChargebackVm).include?(model)
```

Comment:
On constantize on a user string: validate by hash lookup for example. But that is pretty similar to what you did.
I am quite ok with the `chargeback_model?`. I agree with @simon3z that it's far from perfect, but it's better than the `start_with?`, it's a method instead of cut-n-pasting the test and it's ok with Brakeman.